.gitignore

```diff
@@ -33,3 +33,5 @@ nbproject
 Thumbs.db
 var/
 .cache/
+/.ddev/*
```

Comment:
Actually, .ddev should bring its own .gitignore file, so these entries should not be necessary.

Reply:
I suggest to decouple this change from the Makefile.

Reply:
ddev in the newest versions does not bring its own ignore file anymore. and there is really no harm in excluding this.

Reply:
See #458